src/main/java/com/ericsson/eiffel/remrem/shared/MsgService.java

```diff
@@ -4,5 +4,12 @@
 import com.google.gson.JsonObject;
 
 public interface MsgService {
+    
+    String EVENT_ID = "eventId";
```

Comment:
The constants (if needed) belong to remrem-semantics / eiffel3messaging. This library along with remrem-generate and remrem-publish should not have any knowledge of different implementations of protocols.